### PR TITLE
Change Omega2 RGB example to use sliders

### PR DESCRIPTION
**Pull request by Himura2la**
Docs: https://github.com/cloud4rpi/docs/pull/22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ You can use this control panel to switch the [onboard LED](https://docs.onion.io
 
 ![](https://github.com/cloud4rpi/docs/raw/master/example-img/omega_led.png)
```
Comment (sky3d) [inline README.md:43]: You need to change image for a slider.

**Review by sky3d — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Himura2la) [reply README.md:43]: The sliders appear later, it's an onboard LED

Comment (Himura2la) [reply README.md:43]: I added a sliders image

**Review by sky3d — APPROVED** [review]

**Review by const314 — APPROVED** [review]

**Comment by vutenkov** [comment]
> All checks have failed

**Comment by Himura2la** [comment]
That's because Python 3.3 does not have `enum` module, and `pylint` does not even start. We need to replace 3.3 with 3.7

**Comment by vutenkov** [comment]
> https://github.com/cloud4rpi/cloud4rpi-omega2-python/pull/4
> To fix the build: https://github.com/cloud4rpi/cloud4rpi-omega2-python/compare/tavis-python37

**Comment by Himura2la** [comment]
This failure is unrelated to the PR itself, the probability of a real failure in 3.3 or 3.7 is very low.

**Pull request merged**